### PR TITLE
Fixes #475 (bump LTS version to 13.16)

### LINKED ISSUE #475: Switch to LTS-13.6
**Original comment from Kim**

You may also consider bumping the lts, 13.16 does have the right version

_Originally posted by @kim in https://github.com/oscoin/oscoin/pull/474_

**Problem**

`stm-containers` recently added a manually-uploaded cabal revision which broke `cabal new-build`.

**Solution**

Switch to LTS-13.6 which has the right version

**Why do we need this?**

Because otherwise our LTS `cabal.config` and our `cabal.project.freeze` would be divergent, and that's the fastest track to madness.

### PR DESCRIPTION
**Pull request by adinapoli-mndc**

### CODE REVIEW

File: cabal.project.freeze [hunk 1]
```diff
@@ -1,19 +1,26 @@
+-- NOTE: Due to revisions, this file may not work. See:
+-- https://github.com/fpco/stackage-server/issues/232
```
Comment (adinapoli-mndc) [inline cabal.project.freeze]: This is straight copy-pasta from https://www.stackage.org/lts-13.16/cabal.config

**Comment by adinapoli-mndc** [comment]
@kim any clue why the "Build deps" section of CI is not happy? My google-cloud-fu is weak, can the master help?

**Comment by kim** [comment]
Ah well, it wants a newer GHC in the base image

**Comment by adinapoli-mndc** [comment]
@geigerzaehler Jawohl! on it.

**Comment by adinapoli-mndc** [comment]
If my reptilian brain understands how these CI scripts work, we are currently blocked by https://github.com/freebroccolo/docker-haskell/issues/87 💔

**Pull request merged**